### PR TITLE
Zombienet: add cpu/mem resource requests

### PR DESCRIPTION
**Pull request by sandreim**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -718,8 +717,7 @@ zombienet-tests-parachains-disputes:
     - /home/nonroot/zombie-net/scripts/run-test-env-manager.sh
         --github-remote-dir="${GH_DIR}"
         --test="0002-parachains-disputes.feature"
-  allow_failure:                   true
-  retry: 2
```
Comment (ordian) [inline .gitlab-ci.yml:722]: why remove the retries?

Comment (sandreim) [reply .gitlab-ci.yml:722]: To ensure possible issues are not hidden by retries.

Comment (ordian) [reply .gitlab-ci.yml:722]: Well, if the issues are resolved by retries, are they not most likely spurious?

Comment (sandreim) [reply .gitlab-ci.yml:722]: Yeah, but the assumption is now that these shouldn't fail and if it happens,  some level of scrutiny is needed to see if the test is flaky or there is an underlying issue.

**Review by drahnr — APPROVED** [review]
I'd be more comfortable to keep the retries for the time being, but I see your point that we should fix spurious issues rather than shadowing them.
Merge at will

**Review by pepoviola — APPROVED** [review]

**Comment by pepoviola** [comment]
I just retry `zombienet-tests-parachains-pvf`, from the logs the spawn of one of the nodes fails. I will check with @emamihe the infra to know if was some kind of bottleneck.

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -688,8 +688,7 @@ zombienet-tests-parachains-pvf:
     - /home/nonroot/zombie-net/scripts/run-test-env-manager.sh
         --github-remote-dir="${GH_DIR}"
         --test="0001-parachains-pvf.feature"
-  allow_failure:                   true
-  retry: 2
+  allow_failure:                   false
```
Comment (alvicsam) [inline .gitlab-ci.yml:691]: Can be removed, the default value is `false`

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -718,8 +717,7 @@ zombienet-tests-parachains-disputes:
     - /home/nonroot/zombie-net/scripts/run-test-env-manager.sh
         --github-remote-dir="${GH_DIR}"
         --test="0002-parachains-disputes.feature"
-  allow_failure:                   true
-  retry: 2
+  allow_failure:                   false
```
Comment (alvicsam) [inline .gitlab-ci.yml:720]: Can be removed, the default value is `false`

File: .gitlab-ci.yml (hunk 2 above)
Comment (drahnr) [reply .gitlab-ci.yml:691]: I prefer explicit to implicit, that way, if there is a regression, it avoids the detour via the gitlab y'all spec docs

**Comment by sandreim** [comment]
> I just retry `zombienet-tests-parachains-pvf`, from the logs the spawn of one of the nodes fails. I will check with @emamihe the infra to know if was some kind of bottleneck.

This sounds like a good reason to keep the retries until we fix it.

**Comment by sandreim** [comment]
bot merge

**Comment by sandreim** [comment]
bot merge

**Pull request merged**